app/services/conversation-channel.js

```diff
@@ -7,6 +7,11 @@ export default PhoenixSocket.extend({
   session: service(),
   store: service(),
 
+  init() {
+    get(this, 'session').on('authenticationSucceeded', () => this.connect());
```

Comment:
@begedin is this service instantiated when a user enters the ember app and before the authenticationSucceeded event is triggered?

Reply:
Ah nvmd.  This is just passively listening.  `join` will call connect at the right time.